packages/nextjs/src/edge/withSentryAPI.ts

```diff
@@ -0,0 +1,29 @@
+import { getCurrentHub } from '@sentry/core';
+
+import type { EdgeRouteHandler } from './types';
+import { withEdgeWrapping } from './utils/edgeWrapperUtils';
+
+/**
+ * Wraps a Next.js edge route handler with Sentry error and performance instrumentation.
+ */
+export function withSentryAPI<H extends EdgeRouteHandler>(
```

Comment:
l: Could we get some unit tests for this logic? In particular for the dynamic nature of `spanLabel` and `spanOp`

Reply:
Added in 76e4815b145da9cbe2de24256d6868e7636d72a5